src/Event/Subscriber/OmnipayGatewayRequestSubscriber.php

```diff
@@ -0,0 +1,103 @@
+<?php
+/**
+ * payment-gateway-logger
+ *
+ * @package    payment-gateway-logger
+ * @author     manzoj
+ * @version    1
+ */
+
+namespace PaymentGatewayLogger\Event\Subscriber;
+
+use Guzzle\Common\Event;
+use PaymentGatewayLogger\Event\Constants;
+use Psr\Log\LoggerInterface;
+use Psr\Log\LogLevel;
+use Symfony\Component\EventDispatcher\EventSubscriberInterface;
+
+class OmnipayGatewayRequestSubscriber implements EventSubscriberInterface
```

Comment:
I think having this class in this repo makes sense. It's handy, and people can extend it to change the shapes of the logs.